src/causalprog/_abc/labelled.py

```diff
@@ -20,4 +20,4 @@ def label(self) -> str:
         return self._label
 
     def __init__(self, *, label: str) -> None:
-        self._label = str(label)
+        self._label = label
```

Comment:
```suggestion
        self._label = str(label)
```

Looks to have snuck in again?